linux/step06_centos7_daemon.sh

```diff
@@ -1,6 +1,15 @@
 #!/bin/bash
 
-cp omero-systemd.service /etc/systemd/system/omero.service
+PGVER=${PGVER:-pg94}
+
+if [ "$PGVER" = "pg95" ]; then
+	cp omero-pg95-systemd.service /etc/systemd/system/omero.service
+else
```

Comment:
@jburel could you make IF pg95 and IF pg94, we cannot have ELSE, otherwise my trick to skip PG won't work anymore